apps/api/docs/production-setup.md

```diff
@@ -24,19 +24,19 @@ Make sure Redis and MariaDB server are up and running.
   ```env
   # Server
   SERVER_PORT=3000
-  SERVER_API_KEY=osmo-notify-test-key
+  SERVER_API_KEY=osmoX-test-key
```

Comment:
Update to `OsmoX`

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b